### PR TITLE
i18n: use qt6 dir for translations when building on qt6

### PR DESCRIPTION
**Pull request by charliewolf**
hi, i'm working on updating a qownnotes rpm spec to support qt6 and ran into some issues with this hardcoded path.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -49,8 +49,13 @@ void loadTranslations(QTranslator *translator, const QString &locale) {
     loadTranslation(translator[5], appPath + "/languages/QOwnNotes_" + locale);
     loadTranslation(translator[6], appPath + "/QOwnNotes_" + locale);
     loadTranslation(translator[7], "../src/languages/QOwnNotes_" + locale);
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```
Comment (pbek) [inline src/main.cpp:52]: You'll need a `>=` check here, wouldn't you?

Comment (charliewolf) [reply src/main.cpp]: indeed, fixed.

**Comment by pbek** [comment]
Will need changes too:

https://github.com/pbek/QOwnNotes/blob/68c860e109c0e9f99f5541aa48ebec4cc34e3052/src/main.cpp#L62

File: src/main.cpp (hunk 1 above)
Comment (pbek) [reply src/main.cpp]: thank you

**Comment by charliewolf** [comment]
got it, added a check in `loadReleaseTranslations` as well.

**Comment by pbek** [comment]
I tested building on NixOS with Qt 6 and for some reason the `qt6/translations` wasn't even created 🤔 .

**Comment by charliewolf** [comment]
i'm not familiar with nix but this patch worked for me on RHEL 9. did it create the qt5 dir or what did it do?

**Comment by pbek** [comment]
I got it working now. LGTM. Thank you.

**Comment by pbek** [comment]
Hm, seems like a lot of Qt6 builds break...
https://github.com/pbek/QOwnNotes/actions/runs/5887832027

**Comment by barolo** [comment]
> Hm, seems like a lot of Qt6 builds break... https://github.com/pbek/QOwnNotes/actions/runs/5887832027

Hi is support for Qt6  complete now?

**Comment by pbek** [comment]
> Hi is support for Qt6 complete now?

Yes, quite a while ago...

**Comment by barolo** [comment]
> > Hi is support for Qt6 complete now?
> 
> Yes, quite a while ago...

Oh. Would you have some time to update ebuilds then?

**Pull request merged**